airflow/providers/amazon/aws/executors/ecs/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# hadolint ignore=DL3007
+FROM apache/airflow:latest
+USER root
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends unzip \
+  # The below helps to keep the image size down
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+RUN    curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+RUN    unzip awscliv2.zip && ./aws/install
```

Comment:
If you combine these into one step you can toss the zip file so it doesn't sit in a layer. Might also want to toss the expanded files too?

Reply:
Yupp, we could consider that.


Ultimately the goal with this Dockerfile is to provide a very simple and beginner image for folks to play around with. If people are using this in their production workflows, they surely have their own images they're going to build and use all integrated into their CI and pipelines already. So we've gone with the philosophy of not over engineering this one. But this change seems easy/simple enough to make :+1: CC @syedahsn 

I don't think the install_aws.sh script would be available in this image? If it is we should definitely use it!

Reply:
The problem is examples have a tendency to be used as-is in prod, so it's best to be robust if the cost isn't too high, imo.

Reply:
I combined the install process into one layer, and removed the unnecessary files.